srv/GetSimulatorFeatures.srv

```diff
@@ -0,0 +1,32 @@
+# Get a list of features supported by the simulator.
+# Simulators adhering to the standard need to implement at least this interface.
+
+---
+
+uint8 SPAWNING                  = 0      # Supports spawn interface (SpawnEntity).
+uint8 DELETING                  = 1      # Supports deleting entities (DeleteEntity).
+uint8 SPAWN_POSES               = 2      # Supports predefined spawn poses (GetSpawnPoses).
+uint8 SPAWN_BOUNDS              = 3      # Supports spawn bounds (SpawnableBounds).
+uint8 SPAWN_FORMAT_SDF          = 4      # Supports spawning from SDFormat specified in the uri field of SpawnEntity.
+uint8 SPAWN_FORMAT_URDF         = 5
+uint8 SPAWN_FORMAT_USD          = 6
```

Comment:
A) I don't know what USD is other than U.S. Dollar, and this definition doesn't tell me. 
B) Is there a reason we can't just include a list of strings for supported spawn formats?

Reply:
A) Good catch. I'll add an explanation - USD stands for Universal Scene Description
B) Worth considering. I am not sure what else to include on the list (it is not meant to include asset formats such as fbx or stl), could you give some examples?

Reply:
B) I don't know what else would make the list, but if some native format becomes popular in the future, having a list of strings would future-proof this without having to add constants for every new format.

Reply:
Thanks! Added documentation for formats, separated out the message, added string list field for custom formats.